src/fragments/lib-v1/graphqlapi/flutter/authz/2X_add_plugin.mdx

```diff
@@ -1,7 +1,8 @@
 Then, include it, along with any other auth providers, in the call to `addPlugin`.
 
 ```dart
-await Amplify.addPlugin(AmplifyAPI(
+await Amplify.addPlugin(
```

Comment:
I don't think this is formatted quite right. The way it was previously was fine so I would revert this change, but if you wanted to update the formatting to have AmplifyAPI on a new line it should be:

```dart
await Amplify.addPlugin(
  AmplifyAPI(
    authProviders: const [
      CustomOIDCProvider(),
      CustomFunctionProvider(),
    ],
  ),
);
```